addons/web/static/src/search/control_panel/control_panel.xml

```diff
@@ -36,7 +36,7 @@
                     <span class="d-none d-xl-block me-auto"/> <!-- Spacer -->
                 </div>
 
-                <div class="o_control_panel_actions d-empty-none d-flex align-items-center justify-content-start order-2 order-lg-1 w-100 w-lg-auto">
+                <div class="o_control_panel_actions d-empty-none d-flex align-items-center justify-content-start gap-1 order-2 order-lg-1 w-100 w-lg-auto">
```

Comment:
It's unclear to me why this change is needed, could you tell me more about this change?
Screenshots maybe?

Reply:
![image](https://github.com/odoo/odoo/assets/128030743/49353f72-a98e-41cc-a822-a26c8a7865b5)
![image](https://github.com/odoo/odoo/assets/128030743/5fe687c5-2478-41ec-83c2-f3f25e766025)

This fixes the spacing issue in Consolidation > Consolidated Balance

Reply:
I'm facing huge issues with my localhost. Nothing compiles automatically. I have to restart a DB everytime I do a change...

Meanwhile in managed to find the file where these buttons are located to I managed to apply the changed only there